package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-entity-config",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
This shouldn't be updated as part of the PR but can be done as part of a new tagged commit for version update along with an update to the changelog.

Reply:
Okay cool, I guess I leave that to you then?